### ISSUE TITLE
Windows sparkling-shell cannot find version

### ISSUE THREAD

**Issue by NoahMarconi** [h2oai/sparkling-water#97]
Hi, testing out using windows and I'm getting an error after the cmd script fails to find a versions nubmer:

``` cmd
sparkling-shell --conf "spark.executor.memory=1g"
find: 'version': No such file or directory
You are trying to use Sparkling Water built for Spark , but your %SPARK_HOME(=C:/spark-2.0.0-bin-hadoop2.7) property points to Spark of version ~-5. Please ensure correct Spark is provided and re-run Sparkling Water.
```

**Comment by mmalohlava** [comment]
Hi @NoahMarconi ,

thanks for bug report. Spark slightly change CLI and for some reason `--version` expects an additional parameter. It is fix on linux, but we leaked the problem into Windows distrib.

We will fix it in the following SW release.

Thank you!

**Comment by NoahMarconi** [comment]
Sounds good, I'm just kicking the tires at the moment so I can run in a linux vm in the meantime.

**Comment by jakubhava** [comment]
Starting to work on the issue. It's actually related to all spark versions, since there is a bug in windows script. Hopefully will provide fix soon

**Comment by jakubhava** [comment]
problem solved, please this this PR https://github.com/h2oai/sparkling-water/pull/100. Once the review is done the change will be merged into master
